composer.json

```diff
@@ -20,7 +20,7 @@
     "require": {
         "php": "^7.1",
         "slevomat/coding-standard": "^5.0",
-        "squizlabs/php_codesniffer": "3.5.*"
+        "squizlabs/php_codesniffer": "~3.5.0"
```

Comment:
@dereuromark This will pull in `3.6.0` when it's released isn't it? That's what I wanted to prevent and new phpcs minor release almost always cause problems, so it's best of stick with bugfix releases only and manually bump up release.

Reply:
No, semver states ~ as locking to the last digit here.

Reply:
Ah, nvm then.